admin/src/main/java/edu/unc/lib/dl/admin/controller/PerformanceMonitorController.java

```diff
@@ -0,0 +1,299 @@
+	private static final Logger log = LoggerFactory
+			.getLogger(PerformanceMonitorController.class);
+	
+	private static final String NEW_LINE_SEPARATOR = "\n";
+	private static final Object [] FILE_HEADERS = {
+		"date",
+		"uuid",
+		"throughput_files",
+		"throughput_bytes",
+		"queued_duration",
+		"ingest_duration",
+		"finished",
+		"moves",
+		"finished_enhancements",
+		"failed_enhancements",
+		"failed_deposit",
+		"failed_deposit_job"
+	};
+	private CSVFormat csvFileFormat = CSVFormat.DEFAULT.withRecordSeparator(NEW_LINE_SEPARATOR);
+	
+	@Autowired
+	private JedisPool jedisPool;
+	
+	public JedisPool getJedisPool() {
+		return jedisPool;
+	}
+
+	public void setJedisPool(JedisPool jedisPool) {
+		this.jedisPool = jedisPool;
+	}
+
+	public Set<String> getDepositMetrics() {
+		try (Jedis jedis = getJedisPool().getResource()){
+			return jedis.keys("deposit-metrics:*");
+		}
+	}
+	
+	private Boolean buildFile(String path) {
+		File csvFile = new File(path);
+		
+		if(csvFile.exists() && !csvFile.isDirectory()) { 
+			Long currentTime = System.currentTimeMillis();
+			Long fileCreationTime = csvFile.lastModified();
+			
+			if ((currentTime - fileCreationTime ) > 3600000) {
+				return true;
+			}
+			return false;
+		} else if (!csvFile.exists()) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+	
+	/**
+	 * Gets totals for metrics that are only aggregated daily such as moves & enhancements
+	 * It also grabs throughput totals for older data, before these fields were set to measure performance at the individual deposit level
+	 * @return
+	 */
+	public String getOperationsData() {
+		FileWriter fileWriter = null;
+		CSVPrinter csvFilePrinter = null;
+		Set<String> operations = null;
+		String filePath = "/opt/data/ingest-times-daily.csv";
+		Map<String, String> depositJob = null;
+		Map<String, String> operationJob = null;
+		String[] depositKeys = null;
+
+		if (buildFile(filePath)) {
+			Set<String> deposits = getDepositMetrics();
+			try (Jedis jedis = getJedisPool().getResource()) {
+				operations = jedis.keys("operation-metrics:*");
+
+				fileWriter = new FileWriter(filePath);
+				csvFilePrinter = new CSVPrinter(fileWriter, csvFileFormat);
+				csvFilePrinter.printRecord(FILE_HEADERS);
+
+				Boolean matchingDate = false;
+				for (String deposit : deposits) {
+					depositJob = jedis.hgetAll(deposit);
+					depositKeys = deposit.split(":");
+					
+					// Ignore data for individual deposits by uuid. Only need the daily ones in this instance
+					if (depositKeys.length > 2) {
+						continue;
+					}
+
+					String jobDate = depositKeys[1];
+					String throughputFiles = depositJob.get("throughput-files");
+					String throughputBytes = depositJob.get("throughput-bytes");
+					String finished = depositJob.get("finished");
+					String failed = depositJob.get("failed");
+					String failedDepositJob = depositJob.get("failed-job:edu.unc.lib.dl.cdr.services.techmd.TechnicalMetadataEnhancementService");
```

Comment:
The depositJob shouldn't have anything about the TechnicalMetadataEnhancementService in it, that is an enhancement.  I'm guessing you don't actually need to try to pull this here since you are populating enhancement stats here, and judging by ActivityMetricsClient, it doesn't look like we are recording the failed job per deposit currently (but probably should), just per day